utilities/utilities.go

```diff
@@ -76,3 +78,27 @@ func versionHandler(c *gin.Context) {
 	}
 	c.File(versionFile)
 }
+
+func GetVersion() string {
+	versionFilename := os.Getenv("VERSION_FILE")
```

Comment:
I'm not sure this would work. In most cases, outside the docker container the version file is going to be the default stub :/

Reply:
We can probably change than with the makefile: https://github.com/Kinto/kinto/blob/master/Makefile#L76-L81